### DISCUSSION TITLE
Support Page Up and Page Down keys.

### DISCUSSION THREAD

**Discussion by grosch** [git-for-windows/git#3224] [Ideas]
Feature request:  On windows, running gitbash, please make the Page Up and Page Down keys scroll the window appropriately.

**Comment by dscho** [comment]
Are you maybe looking for <kbd>Shift</kbd>+<kbd>PageUp</kbd> and <kbd>Shift</kbd>+<kbd>PageDown</kbd> (see e.g. [here](https://mintty.github.io/mintty.1.html#USAGE) or [here](https://defkey.com/mintty-3-shortcuts#75406))?

**Reply by derrickstolee** [reply]
TIL. I confirmed this works.
